Gemfile

```diff
@@ -1,16 +1,22 @@
 source 'https://rubygems.org'
 
 # Bundle edge Rails instead: gem 'rails', github: 'rails/rails'
-gem 'rails', '5.0.0.1'
+gem 'rails', '5.0.1'
 
 # Turbolinks makes following links in your web application faster. Read more: https://github.com/rails/turbolinks
-gem 'turbolinks'
+gem 'turbolinks', '5.0.1'
 
-# gem 'grom'
-gem 'grom', git: "https://github.com/ukpds/grom", branch: "development"
 gem 'json-ld', '2.1.0'
-gem 'vcard'
-gem 'pugin', path: './pugin'
+gem 'vcard', '0.2.15'
+
+# Parliament Ruby is a wrapper for the internal Parliament data API
+gem 'parliament-ruby', git: 'https://github.com/ukparliament/parliament-ruby'
```

Comment:
@mattrayner @katylouise yes, I'm setting the version to `'0.2.1'` in the new Gemfile on my branch 🙂